src/main/java/ooo/foooooooooooo/velocitydiscord/discord/Discord.java

```diff
@@ -169,6 +185,15 @@ public void sendMessage(String message) {
         activeChannel.sendMessage(message).queue();
     }
 
+    public void sendWebhookMessage(String name, String avatar, String content) {
```

Comment:
(can we make this "avatar, name, content" instead? it just makes more sense to me 😭)